assignment-client/CMakeLists.txt

```diff
@@ -12,7 +12,7 @@ setup_memory_debugger()
 # link in the shared libraries
 link_hifi_libraries(
   audio avatars octree gpu graphics fbx entities
-  networking animation recording shared script-engine embedded-webserver
+  networking deepMotion animation recording shared script-engine embedded-webserver
```

Comment:
From what I can see deepMotion doesn't fit the library naming conventions here, it should be named deep-motion because of that. 
It's a minor thing though, from my perspective.